mifosng-android/src/main/java/com/mifos/objects/client/Client.java

```diff
@@ -98,6 +100,22 @@ public Integer getClientId() {
         return clientId;
     }
 
+    /**
+     * This method is returning the comma separated names of all the client's group
+     * in the form of a string.
+     */
+    public String getGroupNames() {
+        String groupNames = "";
+        if (groups.isEmpty())
+            return "";
+        if (groups.size() == 1)
```

Comment:
Is this condition really required? Won't the next block handle this case as well?

Reply:
This condition will take care that if there is only one Group, then comma(,) wouldn't be appended to its name.

Reply:
`return groupNames.substring(0, groupNames.length() - 2);` won't this take care of that anyway?

Reply:
Oh yes, it will work actually. I will remove that condition. Thanks.